backend/de.metas.acct.base/src/main/sql/postgresql/ddl/functions/ProductsWithoutInitialCost.sql

```diff
@@ -0,0 +1,65 @@
+DROP FUNCTION IF EXISTS ProductsWithoutInitialCost
+    (p_ad_org_id NUMERIC);
+
+CREATE OR REPLACE FUNCTION ProductsWithoutInitialCost(p_ad_org_id NUMERIC)
+    RETURNS table
+            (
+                M_Product_ID numeric,
+                ProductValue text,
+                ProductName text
+            )
+AS
+$$
+WITH element AS
+         (
+             SELECT el.m_costelement_id
+
+             FROM c_acctschema s
+                      JOIN m_costelement el on s.costingmethod = el.costingmethod
+             WHERE s.ad_org_id = 1000000
```

Comment:
```suggestion
             WHERE s.ad_org_id = p_ad_org_id
```

..but please also check my private remark

Reply:
I forgot my testing data in there.  Fixed now.